store/tikv/region_cache.go

```diff
@@ -372,9 +372,11 @@ func (c *RegionCache) GetTiFlashRPCContext(bo *Backoffer, id RegionVerID) (*RPCC
 
 	cachedRegion := c.getCachedRegionWithRLock(id)
 	if cachedRegion == nil {
+		logutil.BgLogger().Info("cache region is nil", zap.String("region_id", fmt.Sprintf("%#v", id)))
```

Comment:
Would it print too many logs in the log file? @lysu How do you think?

Reply:
+1 how about change their log level to debug and we re-enable them when meeting problem?

Reply:
Done.